teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_accessor :specialisation
+
+  def initialize(specialisation)
+    super()
```

Comment:
- [ ] I suggest that you pass the arguments when calling super() method. This is because in the Person class the initialize expects at least 2 arguments . Therefore it would be mandatory to pass `age` in the Teacher initialize and super() method ☺️.

Reply:
ok thx